docs/DeveloperGuide.adoc

```diff
@@ -767,6 +841,78 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+=== Use case: Sync Account
+
+*MSS*
```

Comment:
Spell out the MSS or write in glossary ?

Reply:
Done